### ISSUE TITLE
Suggestions for changes to "How It Works" and "Explore Our Companion Projects" text

### ISSUE THREAD

**Issue by HeatherBrooks** [ODM2/ODM2DataSharingPortal#219]
Not tagging this for the 0.7 restyling milestone, but would like to propose for consideration, changes to the homepage that would move the "about" text and links for EnviroDIY and Leaf Pack Network up, and then replace the EnviroDIY, WikiWatershed, and LPN boxes with expanded text and links to wikiwatershed.org.
![data-portal-home-1](https://user-images.githubusercontent.com/9368629/38640406-2fbbb7e8-3da2-11e8-92af-aab48c40d8fb.JPG)
![data-portal-home-2](https://user-images.githubusercontent.com/9368629/38640420-361e4a38-3da2-11e8-8373-51eba55331b0.JPG)
![new-data-portal-home-3](https://user-images.githubusercontent.com/9368629/38640472-5fdd6a5c-3da2-11e8-84f3-5d5888862bbb.jpg)

**Comment by horsburgh** [comment]
Hi all. Thanks for the suggestion. This is a pretty significant change to the work that @Maurier has already done. We'll consider this for a subsequent release.

**Comment by aufdenkampe** [comment]
I really like the new text that @HeatherBrooks proposed under the EnviroDIY and Leaf Pack Network headers. I agree with @horsburgh that the "Learn More>" link might need to wait for a next release, but it seems to me that modifying the text should be straightforward for @Maurier or @jcaraballo17 to do by cut/paste. It had actually been on my mind to write better text than "Sensor Datasets" and "Experiment Datasets".

@HeatherBrooks, could you paste that updated text here, so that  @Maurier or @jcaraballo17 can copy/paste it? Thanks!

**Comment by HeatherBrooks** [comment]
@aufdenkampe, here you go:

EnviroDIY is a community of enthusiasts sharing do-it-yourself ideas for environmental science and monitoring.

Share and Explore Sensor Datasets

The Leaf Pack Network is an international network of teachers, students, and citizen monitors investigating their local stream ecosystems.

Share and Explore Macroinvertebrate Datasets

**Comment by horsburgh** [comment]
@aufdenkampe - It isn't just cut and paste text. @Maurier is going to have to redo the page to get rid of the boxes at the bottom and add the additional text at the bottom. If you want to wait longer, he can do it. We will need the URLs for the links.

**Comment by aufdenkampe** [comment]
@horsburgh, yes, I agree that we can wait on replacing the boxes at the bottom. 
I think what I'm suggesting is indeed a simple copy/paste of text:
* Under the EnviroDIY logo, replace "Sensor Datasets" with this text:
  * "EnviroDIY is a community of enthusiasts sharing do-it-yourself ideas for environmental science and monitoring."
  * "Share and Explore Sensor Datasets"
* Under the LeafPackNetwork logo, replace "Experiment Datasets" with this text:
  * "The Leaf Pack Network is an international network of teachers, students, and citizen monitors investigating their local stream ecosystems."
  * "Share and Explore Macroinvertebrate Datasets"

**Comment by horsburgh** [comment]
@Maurier - can you quickly make these text changes under the logos for the leaf pack network and envirodiy? We won't do the part about the "Explore the WikiWatershed web toolkit" this time around, but the text changes suggested in the last comment by @aufdenkampe should only take a couple of seconds to implement.

**Comment by Maurier** [comment]
@horsburgh yes, I can make this changes today.

**Comment by horsburgh** [comment]
@Maurier - these look good on the sandbox. I just chatted with Juan and he is getting ready for the release. Adding the tested label on this one.

**Comment by aufdenkampe** [comment]
Looks great! Thanks! Looking forward to seeing this release!
